internal/cryptokit/cgo_go124.go

```diff
@@ -18,5 +18,60 @@ package cryptokit
 #cgo nocallback SHA384
 #cgo noescape SHA512
 #cgo nocallback SHA512
+
+#cgo noescape MD5Write
```

Comment:
Better don't mix this PR with heal-allocation changes. Keep it for a follow-up PR.

Reply:
fixed.